resolvers.ts

```diff
@@ -45,6 +49,41 @@ const TEN_MILLION = 10_000_000;
 
 export const resolvers: Resolvers = {
   Query: {
+    // TODO: both of these resolvers should be removed once we can query NG directly from the FE.
+    adminSamples: async (root, args, context: any, info) => {
```

Comment:
Can we put this in its own folder in the `resolver-functions` folder? I have a PR coming to tidy up the rest of the resolvers in this file too.